baselines/flwr_baselines/publications/fedprox_mnist/model.py

```diff
@@ -0,0 +1,166 @@
+"""CNN model architecutre, training and testing functions for MNIST."""
+
+
+from typing import List, Tuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn.parameter import Parameter
+from torch.utils.data import DataLoader
+
+
+class Net(nn.Module):
+    """Convolutional Neural Network architecture as described in McMahan 2017
+    paper :
```

Comment:
Split this onto two separate lines (with an empty line in between - the first sentence in a docstring should not overflow to the next line).